docs/templates/how-to.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Add a title (using an infinitive like this one)
+sidebarTitle: Optional short title if 'title' is long
+description: A short description (up to ~160 characters) of the article that should make sense out of context (like on a listing page).
+---
+
+<!-- 
+When to use
+  When there is a single outcome a user wants to archieve.
+  When you want to explain how to get to the outcome in ordered steps.
+  https://diataxis.fr/how-to-guides/ 
+
+How to use
+  1. Copy this template into the right directory in /src/docs/.
+  2. Rename it to match the title.
+  3. Replace the following content with your own.
+-->
+
+Introduce the topic in 1 or 2 sentences.
+Include reasons why a person might want to do it
+(the most common use cases).
+Also include what the goal of the task is.
+
+## Before you begin
+
+Include here any terms the reader might need
```

Comment:
I feel this could lead some contributors to mistake. How will they know which terms to define? what's the standard for what's obvious and what's not? We might want to wait until we have a glossary to prompt contributors to define terms. Just an idea though.

Reply:
I'm not sure there's an absolute standard other than things readers might need. I added a little more for explanation, but that does make it longer. :shrug: